dockerize.go

```diff
@@ -155,6 +169,7 @@ func main() {
 	flag.Var(&stdoutTailFlag, "stdout", "Tails a file to stdout. Can be passed multiple times")
 	flag.Var(&stderrTailFlag, "stderr", "Tails a file to stderr. Can be passed multiple times")
 	flag.StringVar(&delimsFlag, "delims", "", `template tag delimiters. default "{{":"}}" `)
+	flag.Var(&headersFlag, "header", "HTTP headers, colon separated. e.g \"Accept-Encoding:gzip\". Can be passed multiple times")
```

Comment:
`-header` might not be intuitive since this option is specifically for `-wait` and only applies to the `http` waits.  

Maybe something like `-wait-http-header` or `-wait-header` and the value as `http:"Accept-Encoding:gzip"` perhaps.  That might be easier to understand that it's specifically for wait and allow for the possibility headers for other protocols.